helm-chart/csi-driver/values.yaml

```diff
@@ -11,6 +11,9 @@ enableMetrics: false
 # default metrics address port
 metricsPort: 8081
 
+# enable encryption for volumes
```

Comment:
```suggestion
# enable encryption for volumes by default
```
I think this means it is enabled by default?

Reply:
Since encryption is disabled by default I meant to write "Enable this variable for encryption on volumes". I could just comment "Set this variable to true for encryption" as it would be more descriptive?